### PR TITLE
Add files for initial release

### PR DESCRIPTION
**Pull request by tjwp**
There are three commits here:
- the 1st copies files without any changes
- the 2nd makes modifications (mostly removing ActiveSupport dependencies) to get things working in the gem
- the 3rd commit is some additional cleanup, mostly around the README

Only the last two commits should need review.

Prime: @jturkel

### CODE REVIEW

File: lib/avro/builder/field.rb [hunk 1]
```diff
@@ -63,7 +63,7 @@ def serialize(reference_state)
           doc: doc,
           default: default,
           aliases: aliases
-        }.compact
+        }.select { |_, v| !v.nil? }
```
Comment (jturkel) [inline lib/avro/builder/field.rb]: Nitpick but you can avoid the negation if you do `reject { |_, v| v.nil? }`

Comment (tjwp) [inline lib/avro/builder/field.rb]: I went this this because it was exactly the ActiveSupport implementation.

**Comment by jturkel** [comment]
Looks good. Are you going to setup travis CI for this?

**Comment by tjwp** [comment]
Yes, it's setup in Travis now. Just one ruby version to start with.

**Comment by tjwp** [comment]
This was closed by b1b5c52994753360b6a5744cea59a7549f865641

**Pull request closed without merging**